golang/pkg/events/events.go

```diff
@@ -1,24 +1,53 @@
 package events
 
 import (
+	"github.com/google/uuid"
 	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/types/known/anypb"
+	timestamppb "google.golang.org/protobuf/types/known/timestamppb"
 )
 
-func ToEvent(event proto.Message, source string, id string) ([]byte, error) {
+type CloudEventOptions struct {
+	Id     string
+	Source string
+	Time   *timestamppb.Timestamp
+}
+
+func ToEvent(event proto.Message, opts CloudEventOptions) ([]byte, error) {
```

Comment:
My unique doubt is if we should force the user to provide a cloudeventoptions, or if we could have there a pointer, but we would need to check if its null or not.
What do you think @CDimonaco ?